app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const userRouter = require('./routes/userRoutes');
+
+const app = express();
+
+app.use(express.json());
+app.use('/api/v1/users', userRouter);
```

Comment:
I'd suggest you to create completely separate router in routes directory:

`const router = express.Router();

router.use('/users', userRouter);

... app.js ...
const router = require('./routes')
app.use('/api/v1', router)
`